crates/server/src/config.rs

```diff
@@ -0,0 +1,87 @@
+use hyper::header::HeaderValue;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::str::FromStr;
+
+/// Additional server options.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ServerConfig {
+    /// The cors `allow_origin` header
+    pub allow_origin: HeaderValueWrapper,
+    /// Whether to enable CORS
+    pub no_cors: bool,
+}
+
+// === impl ServerConfig ===
+
+impl ServerConfig {
+    /// Sets the "allow origin" header for cors
+    pub fn with_allow_origin(mut self, allow_origin: impl Into<HeaderValueWrapper>) -> Self {
+        self.allow_origin = allow_origin.into();
+        self
+    }
+
+    /// Whether to enable CORS
+    pub fn set_cors(mut self, cors: bool) -> Self {
+        self.no_cors = cors;
```

Comment:
Something looks wrong here. Where is this method even used?

Reply:
Removed in [5a51d85](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/8/commits/5a51d85e8c461d1d0b92a3a0b152185f13904aa7)